src/abstractdataframe/iteration.jl

```diff
@@ -7,14 +7,23 @@
 # TODO: Reconsider/redesign eachrow -- ~100% overhead
 
 # Iteration by rows
+"""
+    DFRowIterator{<:AbstractDataFrame}
+
+Iterator over rows of an `AbstractDataFrame`.
+Each returned value is represented as a `DataFrameRow`.
```

Comment:
I'd say "with each row represented as a `DataFrameRow`" (as below), in a single sentence.

Reply:
fixed